api_docs/changelog.md

```diff
@@ -1241,8 +1241,10 @@ field with an integer field `invite_to_realm_policy`.
 
 **Feature level 35**
 
-* The peer_add and peer_remove subscription events now have plural
-  versions of `user_ids` and `stream_ids`.
+* [`GET /events`](/api/get-events): The `subscription` events for
+  `peer_add` and `peer_remove` now return `user_ids` and `stream_ids`
+  arrays. Previously, these events returned singular `user_id` and
```

Comment:
nit: The verb "return" here feels odd to me — I would say the endpoint returns these events, or even returns these arrays inside these events, but the events themselves are just data and don't return anything or otherwise actively do anything. Rather the events "contain" these arrays and contained those integers, or perhaps "provide" or "include".

(I see there are some other examples on this page of an event "returning" the data in it, but not many.)

Reply:
Also made this change in the **Changes** note for both these events.